### ISSUE TITLE
Argus Checkin Force Closes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#21]
```
date    Sat, Aug 8, 2009 at 2:13 PM

From Dean:
"""
Actually I checked into the argus (still the mayor :) and had to force close 
to when shutting down. Not sure if it is repeatable as I am drinking
"""
```

Original issue reported on code.google.com by `jlapenna` on 8 Aug 2009 at 10:56

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 087902289d.
```

Original comment by `jlapenna` on 9 Aug 2009 at 4:14
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 087902289d.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:46
